src/romein.cu

```diff
@@ -1,5 +1,5 @@
 /*
```

Comment:
Are the changes in this file still relevant, or are these the changes that eventually became the other blocks? If the latter, we should revert this to the version that really is Romein.